public/app/features/dashboard/components/GenAI/jsonDiffText.test.ts

```diff
@@ -0,0 +1,431 @@
+import { createDashboardModelFixture, createPanelJSONFixture } from '../../state/__fixtures__/dashboardFixtures';
```

Comment:
seems like CI build is failing due to `createPanelJSONFixture` import cc @ivanortegaalba

Reply:
👀

Reply:
Fixed!